index.html

```diff
@@ -2056,6 +2056,25 @@ <h2 id="AudioContext">
                 "#widl-BaseAudioContext-baseLatency">baseLatency</a> attribute.
               </p>
             </dd>
+            <dt>
+              float sampleRate
+            </dt>
+            <dd>
+              <p>
+                Set the <a href=
+                "#widl-BaseAudioContext-sampleRate"><code>sampleRate</code></a>
+                of the context to this value. The supported values are the same
+                as the sample rates for an <a>AudioBuffer</a>. <span class=
+                "synchronous">A <code>NotSupportedError</code> exception MUST
+                be thrown if the specified sample rate is not supported.</span>
+              </p>
+              <p>
+                If <a href=
+                "#widl-AudioContextOptions-sampleRate"><code>sampleRate</code></a>
+                is not specified, the sample rate of the audio hardware is
```

Comment:
A bit weird to say hardware here. It's possible that it's the system mixer and not the hardware, maybe we should use a broader term ? We use the term "preferred sample-rate" in gecko, fwiw.

Looks good otherwise.